pkg/apis/apps/types.go

```diff
@@ -393,7 +393,7 @@ type RollingUpdateDeployment struct {
 	// Value can be an absolute number (ex: 5) or a percentage of total pods at the start of update (ex: 10%).
 	// Absolute number is calculated from percentage by rounding down.
 	// This can not be 0 if MaxSurge is 0.
-	// By default, a fixed value of 1 is used.
```

Comment:
Drop the defaulting comment from the internal type. Defaults only apply to external types.